anger.js

```diff
@@ -26,15 +26,16 @@ function anger (opts) {
     ? identifier
     : (payload) => get(payload, identifier)
 
-  for (var i = 0; i < clients.length; i++) {
+  for (let i = 0; i < clients.length; i++) {
     clients[i] = new Client(opts.url)
     if (i < opts.senders) {
       senders[i] = clients[i]
     }
   }
 
+  let i = 0
   steed.each(clients, (client, done) => {
-    client.connect({ auth: auth }, done)
+    client.connect({ auth: auth(client, i++) }, done)
```

Comment:
You're right. I _think_ it will be in order, but @mcollina will need to confirm, or provide some insight on how to get the client index